_aggregations/metric/average.md

```diff
@@ -55,3 +68,117 @@ GET opensearch_dashboards_sample_data_ecommerce/_search
   }
 }
 ```
+
+You can use the aggregation name (`avg_taxful_total_price`) as a key to retrieve the aggregation from the response.
+
+## Missing values
+
+You can assign a value to missing instances of the aggregated field. See [Missing aggregations]({{site.url}}{{site.baseurl}}/aggregations/bucket/missing/).
+
+Prepare an example index by ingesting the following documents. Note that the second document is missing a `gpa` value:
+
+```json
+POST _bulk
+{ "create": { "_index": "students", "_id": "1" } }
+{ "name": "John Doe", "gpa": 3.89, "grad_year": 2022}
+{ "create": { "_index": "students", "_id": "2" } }
+{ "name": "Jonathan Powers", "grad_year": 2025 }
+{ "create": { "_index": "students", "_id": "3" } }
+{ "name": "Jane Doe", "gpa": 3.52, "grad_year": 2024 }
+```
+{% include copy-curl.html %}
+
+### Example: Replacing a missing value
+
+Take the average, replacing the missing GPA field with `0`:
+
+```json
+GET students/_search
+{
+  "size": 0,
+  "aggs": {
+    "avg_gpa": {
+      "avg": {
+        "field": "gpa",
+        "missing": 0
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+The response is as follows. Compare to the next example, where missing values are ignored:
```

Comment:
@dwelsch-esi We need a little more context here. Let's rewrite to indicate what this particular response demonstrates to the user (same comment as on line 178 in [this PR](https://github.com/opensearch-project/documentation-website/pull/9420/files)).

Reply:
```suggestion
As shown in the following example response, the aggregation returns the average `gpa` with `0` substituted for the missing value. Compare this to the next example, where missing values are ignored:
```